src/components/objectListing/objectListing.component.tsx

```diff
@@ -149,7 +162,7 @@ const createColumns = (
           editRowEnabled={opts.withObjectEdit}
           inEditMode={table.options.meta?.rowInEditMode === row.id}
           onEditClick={() => table.options.meta?.onEditClick(row.id)}
-          onInfoClick={() => setPanelInfo({ objectType, uid })}
+          onInfoClick={() => setPanelInfo?.({ objectType, uid })}
```

Comment:
I think we should only add the `actionColumn` to the `orderedColumnArray` when `setPanelInfo` is truthy , like we have for `opts.withObjectSelect`